docs/docsite/rst/network/dev_guide/developing_resource_modules_network.rst

```diff
@@ -558,69 +558,69 @@ The following example walks through the integration tests for the ``vyos.vyos.vy
 
 ``test/integration/targets/vyos_l3_interfaces/tests/cli/overridden.yaml``
 
-.. code-block:: yaml
+.. code-block:: yaml+jinja
 
   ---
-  - debug:
-   msg: START vyos_l3_interfaces merged integration tests on connection={{ ansible_connection
-     }}
+  - debug: null
+    msg: START vyos_l3_interfaces merged integration tests on connection={{
+      ansible_connection }}
 
   - import_tasks: _remove_config.yaml
 
   - block:
 
-   - import_tasks: _populate.yaml
+  - import_tasks: _populate.yaml
```

Comment:
Everything in the `block` is now on the same level as `block` itself.

Reply:
doh! hopefully this is sorted now. I stripped out all the empty lines in the block because I found it really hard to read with them in there.